src/ontology/edits/oeo-shared.omn

```diff
@@ -131,18 +131,87 @@ ObjectProperty: <http://purl.obolibrary.org/obo/IAO_0000136>
     
 ObjectProperty: <http://purl.obolibrary.org/obo/RO_0000053>
 
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000118> "has characteristic"
+    
+    Range: 
+        <http://purl.obolibrary.org/obo/BFO_0000020>
+    
     InverseOf: 
         OEO_00010121
     
     
+ObjectProperty: <http://purl.obolibrary.org/obo/RO_0000056>
+
+    Domain: 
+        <http://purl.obolibrary.org/obo/BFO_0000002>
+    
+    Range: 
+        <http://purl.obolibrary.org/obo/BFO_0000003>
+    
+    
 ObjectProperty: <http://purl.obolibrary.org/obo/RO_0000057>
 
+    Domain: 
+        <http://purl.obolibrary.org/obo/BFO_0000003>
+    
+    
+ObjectProperty: <http://purl.obolibrary.org/obo/RO_0000085>
+
+    Domain: 
+        <http://purl.obolibrary.org/obo/BFO_0000004>
+    
+    Range: 
+        <http://purl.obolibrary.org/obo/BFO_0000034>
+    
+    
+ObjectProperty: <http://purl.obolibrary.org/obo/RO_0000086>
+
+    Domain: 
+        <http://purl.obolibrary.org/obo/BFO_0000002>
+    
     
 ObjectProperty: <http://purl.obolibrary.org/obo/RO_0000087>
 
+    Domain: 
+        <http://purl.obolibrary.org/obo/BFO_0000004>
+    
+    Range: 
+        <http://purl.obolibrary.org/obo/BFO_0000023>
+    
     
 ObjectProperty: <http://purl.obolibrary.org/obo/RO_0000091>
 
+    Domain: 
+        <http://purl.obolibrary.org/obo/BFO_0000004>
+    
+    Range: 
+        <http://purl.obolibrary.org/obo/BFO_0000016>
+    
+    
+ObjectProperty: <http://purl.obolibrary.org/obo/RO_0001025>
+
+    Domain: 
+        <http://purl.obolibrary.org/obo/BFO_0000004>
+    
+    Range: 
+        <http://purl.obolibrary.org/obo/BFO_0000004>
+    
+    
+ObjectProperty: <http://purl.obolibrary.org/obo/RO_0002222>
+
+    Domain: 
+        <http://purl.obolibrary.org/obo/BFO_0000003>
+    
+    Range: 
+        <http://purl.obolibrary.org/obo/BFO_0000003>
+    
+    
+ObjectProperty: <http://purl.obolibrary.org/obo/RO_0002233>
+
+    Domain: 
+        <http://purl.obolibrary.org/obo/BFO_0000015>
+    
```

Comment:
Definition: _p has input c iff: p is a process, c is a **continuant**, c is a participant in p, c is present at the start of p, and the state of c is modified during p._ I think, the range `continuant` should be added.